int8 WAITING_PROCEED = 0
int8 MOVING = 1
int8 PAUSED = 2
int8 IDLE = 3

Header header
int8 status
string current_autosequence
uint32 current_point
string[] defined_autosequences
